nucypher/characters/base.py

```diff
@@ -106,6 +106,12 @@ def __init__(self,
             represented by zero Characters or by more than one Character.
 
         """
+
+        #
+        # Operating Mode
+        #
+        if federated_only is False and BlockchainInterface is None:
```

Comment:
Would it be safer to use `if not federated_only`? Of course, it's a boolean, but nevertheless that's how we use it below

Reply:
yes, and also there's a bug there, since `BlockchainInterface` is the class, not the arg!
I fixed this before, but I lost it in a rebase. I'll fix it now. Good eye!